smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/MediatorManager.java

```diff
@@ -42,6 +44,10 @@
 
     private final List<AbstractMediator> mediators = new ArrayList<>();
 
+    @Inject
+    @ConfigProperty(name = "smallrye.messaging.emitter.default-buffer-size", defaultValue = "127")
```

Comment:
great to see this :) will it need extra property in quarkus as well? or can this be somehow reused/aliased?

Reply:
No, you will be able to use it directly. So you can configure it from the `application.properties` file.